### DISCUSSION TITLE
RocektMQ 5.x Proxy 单独 cluster 部署的情况下需要额外用 nginx 做负载均衡吗

### DISCUSSION THREAD

**Discussion by tyfprof** [apache/rocketmq#7141] [Q&A]
RocketMQ5.x  单独 cluster 部署的情况下需要额外用 nginx 做负载均衡吗? 谢谢！

**Comment by francisoliverlee** [comment]
不行喔， 5.X的客户端从哪个proxy拉取的消息，需要ack给对应的proxy

**Reply by tyfprof** [reply]
你好！
感谢你的答复。
追加一个问题，如果单独部署 proxy cluster，它会自动负载均衡或者具备 HA 能力吗？
